Makefile

```diff
@@ -27,7 +28,7 @@ docker:
 
 .PHONY: docker-run
 docker-run: docker ## Build and run the application in a local docker container
-	@docker run -p ${DEFAULT_APP_PORT}:${DEFAULT_APP_PORT} $(CMD_NAME):latest
+	@docker run -p ${DEFAULT_APP_PORT}:${DEFAULT_APP_PORT} ${ENV_VARIABLES} $(CMD_NAME):latest
```

Comment:
Passing the specified `ENV_VARIABLES` into the docker container allows us to use AWS SSO in our terminal to authenticate and test locally against our sandbox (or whatever) cluster. This avoids us having to provide static client credentials every time during initial testing phase.

Passing it in via `-e` flag is the only way I know how to do it, but maybe there's a better way?

Reply:
This is fine.  An alternative is specifying an `--env-file`.  The Makefile keeps this fairly abstract from the dev.